addons/website_event_booth/controllers/event_booth.py

```diff
@@ -18,9 +18,13 @@ def event_booth_main(self, event, booth_category_id=False, booth_ids=False):
             raise Forbidden()
 
         booth_category_id = int(booth_category_id) if booth_category_id else False
+        menu = request.env["website.menu"].search([('url', '=', request.httprequest.path)])
+        view = request.env["website.event.menu"].sudo().search([('event_id', '=', event.id), ('menu_id', '=', menu.id)]).view_id
+        page = view.key if view else 'website_event_booth.event_booth_registration'
```

Comment:
I don't think this makes any sense (stop me if it does though)

In the "generic" route (/page/<path:page>) we do this in case a custom page was added. But here we know it's not a custom page, since it's a specific URL.

So I would say if we don't find anything, we just don't pass an seo_object.